subsys/net/lib/nrf_provisioning/src/nrf_provisioning.c

```diff
@@ -562,6 +563,26 @@ int nrf_provisioning_req(void)
 			dm.cb(NRF_PROVISIONING_EVENT_STOP, dm.user_data);
 		}
 
+		if (ret == -EACCES) {
+			if (IS_ENABLED(CONFIG_NRF_PROVISIONING_PRINT_ATTESTATION_TOKEN)) {
+				struct nrf_attestation_token token = { 0 };
+				int err;
+
+				err = modem_attest_token_get(&token);
+				if (err) {
+					LOG_ERR("Failed to get token, err %d", err);
+				} else {
+					printk("\nAttestation token "
+					       "for claiming device on nRFCloud:\n");
+					printk("%.*s.%.*s\n\n", token.attest_sz, token.attest,
+					       token.cose_sz, token.cose);
+					modem_attest_token_free(&token);
+				}
+			} else {
+				LOG_WRN("Unauthorized access: device is not yet claimed.");
```

Comment:
Maybe print this always, and print it before the `CONFIG_NRF_PROVISIONING_PRINT_ATTESTATION_TOKEN`

That way if `modem_attest_token_get` fails, we still print something cogent for the user